contrib/docker/docker/build.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-# This script generates a docker image from the latest aspect development version.
+# This script generates a docker image from the latest aspect development version
+# for the linux/amd64 and linux/arm64 architectures.
 # It requires a docker installation on the local machine, and the ability to
 # communicate with the docker daemon without root user privileges (see the docker
 # webpage for an explanation).
 # Note: This container is build from the developer version on Github, it does not use
 # the local ASPECT folder. Therefore local changes are not included in the container.
 
-docker build --no-cache -t geodynamics/aspect .
+docker buildx build --no-cache --platform linux/amd64,linux/arm64 -t geodynamics/aspect .
```

Comment:
interesting. This will cross-compile the arm build on an x64 machine and works on github actions? Nice.

Reply:
Yes, it takes forever though (3x longer than for amd64).